toolkit/tools/pkg/imagecustomizerlib/imagecustomizer_test.go

```diff
@@ -267,70 +244,6 @@ func TestCustomizeImageKernelCommandLineAdd(t *testing.T) {
 	assert.True(t, linuxCommandLineRegex.Match(grub2ConfigFile))
 }
 
-func createFakeEfiImage(buildDir string) (string, error) {
-	var err error
-
-	err = os.MkdirAll(buildDir, os.ModePerm)
-	if err != nil {
-		return "", fmt.Errorf("failed to make build directory (%s):\n%w", buildDir, err)
-	}
-
-	// Use a prototypical Azure Linux image partition config.
-	diskConfig := imagecustomizerapi.Disk{
-		PartitionTableType: imagecustomizerapi.PartitionTableTypeGpt,
-		MaxSize:            4 * diskutils.GiB,
-		Partitions: []imagecustomizerapi.Partition{
-			{
-				Id:    "boot",
-				Type:  imagecustomizerapi.PartitionTypeESP,
-				Start: 1 * diskutils.MiB,
-				End:   ptrutils.PtrTo(imagecustomizerapi.DiskSize(9 * diskutils.MiB)),
-			},
-			{
-				Id:    "rootfs",
-				Start: 9 * diskutils.MiB,
-				End:   nil,
-			},
-		},
-	}
-
-	fileSystems := []imagecustomizerapi.FileSystem{
-		{
-			DeviceId: "boot",
-			Type:     "fat32",
-			MountPoint: &imagecustomizerapi.MountPoint{
-				Path:    "/boot/efi",
-				Options: "umask=0077",
-			},
-		},
-		{
-			DeviceId: "rootfs",
-			Type:     "ext4",
-			MountPoint: &imagecustomizerapi.MountPoint{
-				Path: "/",
-			},
-		},
-	}
-
-	rawDisk := filepath.Join(buildDir, "disk.raw")
-
-	installOS := func(imageChroot *safechroot.Chroot) error {
-		// Don't write anything for the OS.
-		// The createNewImage function will still write the bootloader and fstab file, which will allow the partition
-		// discovery logic to work. This allows for a limited set of tests to run without needing any of the RPM files.
-		return nil
-	}
-
-	err = createNewImageWithBootLoader(rawDisk, diskConfig, fileSystems, "efi", imagecustomizerapi.SELinux{},
```

Comment:
seems createNewImageWithBootLoader is not deleted, do you want to keep it for future use?